integration_tests/utils.py

```
@@ -33,7 +33,7 @@
 from eth_contract.utils import get_initcode
 from eth_contract.utils import send_transaction as send_transaction_async
 from eth_contract.weth import WETH
-from eth_utils import to_checksum_address
+from eth_utils import to_checksum_address, to_hex
```

Ruff (0.12.2)

`eth_utils.to_hex` imported but unused

Remove unused import: `eth_utils.to_hex`

(F401)